### ISSUE TITLE
[bug] kalis version of apktool not recognized correctly

### ISSUE THREAD

**Issue by Lem** [sensepost/objection#444]
**Describe the bug**
With Kali updated to the latest version, apktool will be installed in version 2.5.0. Sadly the _parse_version_ function from _pkg_resources_ seems to have trouble recognizing the provided version by apktool as newer. Therefor the process will be aborted.

**To Reproduce**
1. Upgrade to latest kali
2. Upgrade/Install apktool
3. Run patchapk 'objection patchapk --source orginal.apk '

**Similar issues**
N/A

**Expected behavior**
Version _2.5.0-dirty_ should be recognized as a newer as 2.4.1.

**Evidence / Logs / Screenshots**
Any output from objection, such as stack traces or errors that occurred. Be sure to run objection with the `--debug` flag so that errors from the agent are verbose enough to debug. For example:

```text
user@kali:~/$ objection --debug patchapk --source BLA.apk 
No architecture specified. Determining it using `adb`...
Detected target device architecture as: arm64-v8a
Using latest Github gadget version: 14.2.13
Patcher will be using Gadget version: 14.2.13
Detected apktool version as: 2.5.0-dirty
apktool version should be at least 2.4.1
Please see the following URL for more information: https://github.com/sensepost/objection/wiki/Apktool-Upgrades
apktool is not ready for use
Cleaning up temp files...
```

```text
user@kali:~/$ apt info apktool
Package: apktool
Version: 2.5.0+dfsg.1-2
Priority: optional
Section: devel
[...]
```

```text
user@kali:~/$ apktool -v
Picked up _JAVA_OPTIONS: -Dawt.useSystemAAFontSettings=on -Dswing.aatext=true
Apktool v2.5.0-dirty - a tool for reengineering Android apk files
with smali v2.4.0-debian and baksmali v2.4.0-debian
Copyright 2010 Ryszard Wiśniewski <brut.alll@gmail.com>
[...]
```

**Environment (please complete the following information):**
 - Device: [N/A]
 - OS: [Kali]
 - Frida Version [14.2.13]
 - Objection Version: [1.10.1]

**Application**
N/A

**Additional context**
N/A

**Comment by leonjza** [comment]
I tried to find out what exactly the `-dirty` tag actually means, and resorted to asking [instead](https://twitter.com/leonjza/status/1372454255848742912). One route we can take is to just strip `-dirty`, but I would love to know what it means first.

**Comment by leonjza** [comment]
Ok based on [this](https://twitter.com/iBotPeaches/status/1372489148234412033?s=20) response we can just strip out `-dirty` from the version output.

**Comment by leonjza** [comment]
Should be fixed in #449, and part of a tagged release later today.
